desktop.css

```diff
@@ -1,5 +1,6 @@
-
-*::after ::before {
+/* stylelint-disable csstree/validator */
+/* stylelint-disable declaration-colon-newline-after */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `stylelint`, it will be nice to fix the issue.